meta/library/draft-1.json

```diff
@@ -89,28 +106,65 @@
             "protein expression"
           ]
         },
-        "link to paper": {
+        "Ontology type": {
+          "type": "string",
+          "description": "Type of onthology (TODO: Have an official list of ontologies)",
+          "enum":[
+            "pathway",
+            "cellular component",
+            "molecular component",
+            "biological process"
+          ]
+        },
+        "Organism": {
           "type": "string",
-          "format": "uri",
-          "description": "The link to the paper where this was published"
+          "pattern": "[A-Z][a-z]+ [a-z]+",
+          "description": "Scientific name of the source organism of the library"
         },
-        "link to resource": {
+        "Perturbation Type": {
           "type": "string",
-          "format": "uri",
-          "description": "The link to the resource"
+          "description": "The type of perturbation"
         },
-        "pmid": {
+        "PMID": {
           "type": "string",
           "pattern": "^([0-9]+(; *)?)+$",
```

Comment:
I'm thinking we should update PMID to use an array instead of a semi-colon separated string